src/plugins/dam/screens/search-assets/delegate.js

```diff
@@ -7,6 +7,15 @@ class Delegate extends AbstractDelegate {
       schemas,
     }, dependencies);
   }
+
+  getSearchParams() {
+    const { searchNodesRequestState: { request }, statuses } = this.component.props;
+
+    return {
+      statuses,
```

Comment:
@hunyoboy what about sort here?

Reply:
@gdbrown i added the sort param

Reply:
@hunyoboy don't destruct enums, they are not plain ole objects